<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page1</class>
 <widget class="QDialog" name="Page1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>输入学生成绩</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_subject">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>45</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请输入学生成绩，如果已存在，则会覆盖原成绩</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_grade">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>470</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>学期：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>班级（中文）：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_class">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>470</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>百分制成绩：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_back">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>550</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>72</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>科目：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>72</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>学号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>学分：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_enter">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>550</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>108</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string> 姓名：     </string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_semester">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>112</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <item>
    <property name="text">
     <string>2017秋季学期</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2018春季学期</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_credit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
